attribute/yotiattribute.go

```diff
@@ -0,0 +1,78 @@
+package attribute
+
+import (
+	"log"
+
+	"github.com/getyoti/yoti-go-sdk/anchor"
+)
+
+// Attribute represents a small piece of information about a Yoti user such as a photo of the user or the
+// user's date of birth.
+type Attribute struct {
+	Name    string
+	Value   []byte
+	Type    AttrType
+	Anchors []*anchor.Anchor
+}
```

Comment:
After some thought, I think using the actual attribute proto is the better bet.

Reply:
It would be good to not create a wrapper attribute, but it does make it a lot harder to test. If I have this wrapper, I can create attributes in a clear way in the tests, and test they perform as expected.